packages/shared/friends/selectors.ts

```diff
@@ -36,6 +38,10 @@ export const getConversations = (
     }))
 }
 
+/**
+ * Get all current conversations with messages the user has including DMs, channels, etc
+ * @return `conversation` & `unreadMessages` boolean that indicates whether the conversation has unread messages.
+ */
 export const getAllConversationsWithMessages = (
```

Comment:
We are not referencing this anywhere. Should we keep it? What do you think reviewer? 😛

Reply:
🔪 kill it